src/cocotb/_test.py

```diff
@@ -224,11 +232,7 @@ def create_task(
 
     .. versionadded:: 1.6
     """
-    if isinstance(coro, Task):
-        if name is not None:
-            coro.set_name(name)
-        return coro
-    elif isinstance(coro, Coroutine):
+    if isinstance(coro, Coroutine):
```

Comment:
Why remove `Task` support? That would be a breaking change for https://docs.cocotb.org/en/stable/library_reference.html#cocotb.create_task

Reply:
Yeah I suppose it is, IDR why I changed this, it was a while ago.